src/helpers/folder_helpers.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Helper methods for dealing with directories.
+
+
+# Changes into the given directory
+function change_directory {
+  local directory=$1
+  pwd
+  run_git_command "cd $directory"
+  pwd
+}
+
+
+function undo_steps_for_change_directory {
+  echo "change_directory $(pwd)"
+}
+
+
+# Returns whether the current working directory
+# is the root directory of the current Git workspace.
+function is_in_git_sub_directory {
```

Comment:
Based on the comment this should be named is in `is_in_git_root_directory`. To make use of that may need to implement `echo_if_false`. I think i'd prefer that.